.github/workflows/main.yml

```diff
@@ -30,14 +30,17 @@ jobs:
             python: pypy-3.7
           - os: macos-latest
             python: "3.10"
-          # XXX: We should be testing 3.10, but for some reason setuptools can't
-          # seem to find the runner's C compiler for new wheel builds...
           - os: windows-latest
-            python: "3.9"
+            python: "3.10"
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python }}
       uses: actions/setup-python@v2
+      if: "!endsWith(matrix.python, '-dev')"
+      with:
+        python-version: ${{ matrix.python }}
+    - uses: deadsnakes/action@v2.1.1
```

Comment:
Shouldn’t this one also have a name field?

Reply:
It does not matter as the CI title already shows the Python version.